CHANGELOG.md

```diff
@@ -0,0 +1,66 @@
+# NextEuropa Platform change log
+
+## Version 2.3.0
+
+### Improvements
+  * NEPT-4 - Upgrade - chr module to version 7.x-1.8 (2015-Dec-04)
+  * NEPT-36 - Performance - Remove css and js injectors from the platform codebase
+  * NEPT-61 - DGT connector - Add website identifier in the title of the job sent via poetry
+  * NEPT-62 - DGT connector - Make job deadline field mandatory & do not suggest any date
+  * NEPT-69 - DGT connector - Add behat scenario to test allowing to resend a rejected job with an increment in version
+  * NEPT-71 - DGT connector - Add COUNTER scenarios into behat tests
+  * NEPT-93 - Performance/Varnish - Log into watchdog when web frontend caches are cleared
+  * NEPT-98 - DGT connector - Allow to add more languages to an ongoing translation request
+  * NEPT-112 - Contextual links are not rendered correctly in beans when listed via Views d.o/node/2084823
+  * NEPT-124 - Upgrade - SA-CONTRIB-2016-050 - Flag
+  * NEPT-130 - Upgrade - Display suite to 7.x-2.14 (2016-Apr-26)
+  * NEPT-145 - Tracked changes - Icons shoud always be active when tracked changes exist. (code fix & add test)
+  * NEPT-161 - Upgrade - menu_token to beta7 d.o/node/2838033
+  * NEPT-163 - EU Login - Replace ECAS labels with new labels to align to EU login
+  * NEPT-165 - Ensure PHP 7 compatibility of core platform
```

Comment:
I'm not sure the "core" term is clear for everyone. Here it means the platform without contrib code. But we are the core team and I'm affraid the core word is easily associated with the platform itself by some people.

So I would change this sentence to avoid someone thinking that the platform is fully prepared for PHP 7.